Style/SignUp.css

```diff
@@ -31,15 +31,26 @@
 	vertical-align: top;
 }
 
+#signUpPage .input-box {
```

Comment:
you caould change this to `#signUpPage input ` and that would style the date input boxes too